### PR TITLE
add blurb on disabling endpoint prefix

### PR DESCRIPTION
**Pull request by lucix-aws**
Adds instructions on how to disable endpoint prefixes being applied, inspired by an internal conversation on the topic.

### CODE REVIEW

File: docs/404.html [hunk 1]
```diff
@@ -23,7 +23,7 @@
 <meta property="og:title" content="404 Page not found" />
 <meta property="og:description" content="The AWS SDK for Go V2 provides APIs and utilities that developers can use to build Go applications that use AWS services." />
 <meta property="og:type" content="website" />
-<meta property="og:url" content="http://localhost:1313/aws-sdk-go-v2/404.html" />
```
Comment (Madrigal) [inline docs/404.html:26]: lol

File: docs/404.html [hunk 2]
```diff
@@ -99,7 +99,7 @@
   aria-label="Search this site…"
   autocomplete="off"
   
-  data-offline-search-index-json-src="/aws-sdk-go-v2/offline-search-index.30b245754e63db1f94dcbaa3ba0f9915.json"
+  data-offline-search-index-json-src="/aws-sdk-go-v2/offline-search-index.1ed5201c9429d6d49ab1e6d6c5e5409e.json"
```
Comment (Madrigal) [inline docs/404.html:102]: Were you able to confirm if this offline search works?

**Review by Madrigal — APPROVED** [review]

File: docs/404.html (hunk 2 above)
Comment (lucix-aws) [reply docs/404.html:102]: I did actually check this and it worked for me both in the preview _and_ on the current render of the public site. We'll see what happens when this merges and gets propagated. The search index appears to get a new filename every time, it's possible that it just went out of sync in a previous pass on re-generating.

Comment (lucix-aws) [reply docs/404.html:102]: cc @stefansundin

**Pull request merged**